Wikipedia/Code/ActionHandlerScript.swift

```diff
@@ -26,7 +26,6 @@ final class PageContentService   {
             var textSizeAdjustmentPercentage: String // string like '125%'
             
             var userGroups: [String]
-            var areEditButtonsHidden: Bool
```

Comment:
Just a heads up I made this [task](https://phabricator.wikimedia.org/T252218) to update the bundled files one more time since it seems like the output will be changing once the patch is merged. If I do a search for `areEditButtonsHidden` in the project I see that it's still in pcs.js. These bundled files are referenced and copied into our caching system when the user migrates old saved articles, to allow them to read saved articles when they launch 6.6 in offline mode. We can do this work separate from this PR though.

Reply:
Sounds good. It's probably good to keep them in sync. Though unless you can edit in offline mode, these changes won't affect the offline experience.

Reply:
@mcleinman true, nope you can't edit in offline mode.